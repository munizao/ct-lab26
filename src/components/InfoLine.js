import React from 'react';
import styles from './App.css';
import PropTypes from 'prop-types';

const InfoLine = (props) => (
  <div className={styles.InfoLine}>
    <span>{props.lineHead}</span><p>{props.value}</p>
  </div>
);

InfoLine.propTypes = {
  lineHead: PropTypes.string,
  value: PropTypes.string
};

export default InfoLine;
