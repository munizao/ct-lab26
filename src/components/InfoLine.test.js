import React from 'react';
import { shallow } from 'enzyme';
import InfoLine from './InfoLine';

describe('InfoLine component', () => {
  it('renders InfoLine', () => {
    const wrapper = shallow(<InfoLine lineHead="STATUS" value="Alive" />);
    expect(wrapper).toMatchSnapshot();
  });
});
