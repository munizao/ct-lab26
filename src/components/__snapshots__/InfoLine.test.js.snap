// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoLine component renders InfoLine 1`] = `
<div
  className="InfoLine"
>
  <span>
    STATUS
  </span>
  <p>
    Alive
  </p>
</div>
`;
